scripts/versions_manager.py

```diff
@@ -374,10 +374,12 @@ def __init__(self, target_path="./target", source_path='.'):
     def load_from_target(self):
         dockers_path = os.path.join(self.target_path, 'versions/dockers')
         build_path = os.path.join(self.target_path, 'versions/build')
+        default_path = os.path.join(self.target_path, 'versions/default')
```

Comment:
>default [](http://example.com/codeflow?start=64&length=7)

This directory name is not easy to understand. What does it mean? Do you have alternative names?

Reply:
default versions is designed to record versions in a buster-amd64 build.
other versions will be recorded in another file when they are different with default versions.
For docker image we can use one file to record versions. that is default.